lib/encoder_unittest.py

```diff
@@ -412,6 +413,15 @@ def testInit(self):
     cache = encoder.EncodingDiskCache(StorageOnlyContext())
     self.assertTrue(cache)
 
+  def testSearchPath(self):
+    # The score path starts out empty from setUp.
+    cache = encoder.EncodingDiskCache(StorageOnlyContext())
+    path = cache.SearchPath()
+    self.assertEquals(1, len(path))
```

Comment:
I don't understand this test. Why should the path be 1 and then two after being overridden? Remember you can add msg="" to asserts to better explain the intent of the assert.